### DISCUSSION TITLE
Menu : Allow click event for headers

### DISCUSSION THREAD

**Discussion by DeVanMipih** [primefaces/community#573] [PrimeNG] [answered]
Actually, specifying _command_ for menuItems with children is not effective.
As for regular items, I would like add _command_ or _routerLink_ on headers menuItem.
I know that an issue was created about the possibility of [templating headers](https://github.com/orgs/primefaces/discussions/508), but I 'm seeking for a solution or workaround to do this just by specifying a _command_ in my model.
Have you any tips about this ?

**Comment by mehmetcetin01140** [comment, marked as answer]
Hi,

You can do it by itemTemplate.
